packages/mongo/oplog_tests.js

```diff
@@ -180,6 +182,116 @@ process.env.MONGO_OPLOG_URL && Tinytest.addAsync(
   },
 );
 
+let defaultOplogHandle = MongoInternals.defaultRemoteCollectionDriver().mongo._oplogHandle;
+
+async function oplogOptionsTest({
+  test,
+  includeCollectionName,
+  excludeCollectionName,
+  mongoPackageSettings = {}
+}) {
+  if (!Meteor.settings.packages) Meteor.settings.packages = {};
+  if (!Meteor.settings.packages.mongo) Meteor.settings.packages.mongo = mongoPackageSettings;
```

Comment:
It'd be good to clear those settings after the test not to influence others.

Reply:
I am actually doing just that?! Please just look at the very bottom of `oplogOptionsTest()` :)

Reply:
No, you _delete_ it - I suggest _resetting_ it to what it was. I know that right now it will be the same, but it may change in the future.

Reply:
Also fixed!